docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1347 @@
+{
```

Comment:
<blockquote>All measurements should be terminal and in a single moment.</blockquote><p>One liner explanation for this as well? Maybe because measurements are the slowest operations on the device? and we don't yet support non-terminal measurements?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Added!</p>